sktime/transformations/panel/catch22wrapper.py

```diff
@@ -126,8 +126,6 @@ def __init__(
         else:
             raise ValueError("features must be a str, list or tuple")
 
-        self._transform_features = None
```

Comment:
this is concerning the other estimator - can you explain why you are changing this?
You are probably right, this can be removed.

Even if, it would be better if you could split this out in a separate pull request? The changes seem independent enough, so that's better for testing, review by core developers, and for the release changelog.

Reply:
I didn't see where it is used, so I wanted to remove the additional `if not transform_feature[f_count]: continue` if condition when I was changing the implementation, but I can bring it back of course

Reply:
you're probably right in removing it, I wondered what it was doing, closer inspection reveals, probably nothing. Still, it's easy to miss in a PR focusing on sth else, that's why I suggest simply moving this to another PR.

Reply:
sure will do

Reply:
Brought back in https://github.com/sktime/sktime/pull/6002/commits/1b04ebcacd6dc4f386f6fd4d075d9ad9342f3929